modules/standard/Sys.chpl

```diff
@@ -437,11 +439,12 @@ module Sys {
   extern proc sys_accept(sockfd:fd_t, ref add_out:sys_sockaddr_t, ref fd_out:fd_t):err_t;
   extern proc sys_bind(sockfd:fd_t, ref addr:sys_sockaddr_t):err_t;
   extern proc sys_connect(sockfd:fd_t, ref addr:sys_sockaddr_t):err_t;
-  extern proc sys_getaddrinfo(node:c_string, service:c_string, ref hints:sys_addrinfo_t, ref res_out:sys_addrinfo_ptr_t):err_t;
+  extern "getaddrinfo" proc sys_getaddrinfo(node:c_string, service:c_string, ref hints:sys_addrinfo_t, ref res_out:sys_addrinfo_ptr_t):err_t;
```

Comment:
Why have it named `sys_getaddrinfo` in Chapel? Just name it `getaddrinfo` and then you don't need the renaming.  `extern proc getaddrinfo`